### DISCUSSION TITLE
Blacklist Feature for /itemname

### DISCUSSION THREAD

**Discussion by MasonX3** [EssentialsX/Essentials#4044] [Ideas and Feature Suggestions] [answered]
I'd like to bring a certain feature into consideration.
Blacklisting certain items from being renamed, and possibly strings of text from being named. 

My reasoning behind this idea is that I believe it would be a great tool for community servers.
Smaller survival servers don't exactly have too many plugins or perks for people to donate for, therefore making item-naming one of the few options that are available from essentials without it being absolutely game-breaking like /gamemode. 

A lot of times these smaller servers don't have full knowledge of how to flag items as special, such as player heads or totems of undying from being renamed as some sort of reward.

I'm unsure of how many people would really use it; however, I feel like it could be something that could possibly be helpful in the future.

prevent-type permission

**Comment by JRoy** [comment, marked as answer]
Will be added in #4133
